pyproject.toml

```diff
@@ -1,7 +1,70 @@
 [build-system]
-requires = ["setuptools"]
+requires = ["setuptools", "wheel"]
 build-backend = "setuptools.build_meta"
 
+
+[project]
+name = "linode_api4"
+authors = [{ name = "Linode", email = "developers@linode.com" }]
+description = "The official python SDK for Linode API v4"
+readme = "README.rst"
+requires-python = ">=3.8"
+keywords = [
+    "akamai",
+    "Akamai Connected Cloud",
+    "linode",
+    "cloud",
+    "SDK",
+    "Linode APIv4",
+]
+license = { text = "BSD-3-Clause" }
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Topic :: Software Development :: Libraries",
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
```

Comment:
Would it make sense to also add Python 3.12 here?

Reply:
Just added, thanks!